models/UserGroupRelation.ts

```diff
@@ -0,0 +1,29 @@
+import { Schema, model, models } from 'mongoose';
+
+const UserGroupRelationSchema = new Schema({
+    groupID: {
+        type: String,
+        ref: 'Group'
+    },
+    email: {
```

Comment:
Actually, you are referencing User, then that is `userID`, not `email`

Reply:
Will we use email or user._id for user identifier? cuz I see user model set email unique. Then I guess in the relation, email should be used instead of the userID, avoid naming confusion.

Reply:
> Actually, you are referencing User, then that is `userID`, not `email`

Will we use email or user._id for user identifier? cuz I see user model set email unique. Then I guess in the relation, email should be used instead of the userID, avoid naming confusion.